shared/teams/common/channel-hooks.tsx

```diff
@@ -16,14 +16,24 @@ function filterNull<A>(arr: Array<A | null>): Array<A> {
 export const useAllChannelMetas = (
   teamID: Types.TeamID,
   dontCallRPC?: boolean
-): {channelMetas: Map<ChatTypes.ConversationIDKey, ChatTypes.ConversationMeta>; loadingChannels: boolean} => {
+): {
+  channelMetas: Map<ChatTypes.ConversationIDKey, ChatTypes.ConversationMeta>
+  loadingChannels: boolean
+  triggerReload: () => void
+} => {
   const getConversations = Container.useRPC(RPCChatTypes.localGetTLFConversationsLocalRpcPromise)
 
   const teamname = Container.useSelector(state => Constants.getTeamNameFromID(state, teamID) ?? '')
   const [inboxUIItems, setConvs] = React.useState<RPCChatTypes.InboxUIItem[] | null>(null)
+  const [reloadValue, setReloadValue] = React.useState(0)
+  const triggerReload = () => setReloadValue(reloadValue + 1)
```

Comment:
I think cleaner for this to be a `useCallback` of the content of the `if (!dontCallRPC)` below, and call this there.